loleaflet/src/layer/tile/ScrollSection.ts

```diff
@@ -383,6 +383,11 @@ class ScrollSection {
 			this.sectionProperties.mouseIsOnVerticalScrollBar = false;
 			this.sectionProperties.mapPane.style.cursor = this.sectionProperties.defaultCursorStyle;
 			this.fadeOutVerticalScrollBar();
+
+			// just in case if we have blinking cursor visible
+			// we need to change cursor from default style
+			if (this.map._docLayer._cursorMarker)
+					this.map._docLayer._cursorMarker.setMouseCursor();
```

Comment:
A bit of nitpicking, here and in line 416 there is an extra tab.

Apart from these, I can see the correct pointer, thanks!